index.js

```diff
@@ -70,6 +78,83 @@ function MediaSession(opts) {
 }
 
 
+function offerAnswerQueue(self, errorMsg, jingleDesc, cb) {
+  self.q.push(function(qCb) {
+    function done(err, answer) {
+      qCb(err);
```

Comment:
Does the error needs to be passed to the queue callback?

Reply:
Would we not want both the answer and the error in the callback?